release-notes/docs/3.10.3.md

```diff
@@ -0,0 +1,28 @@
+# 3.10.3 Release Notes
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://docs.communityhealthtoolkit.org/core/overview/supported-software/) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+## Upgrade notes
+
+For full Webapp support for contacts of hardcoded types, that also have a `contact_type` property, ensure your configuration is deployed using [medic-conf 3.4.1](https://github.com/medic/medic-conf/blob/master/release-notes.md#340) or greater.
```

Comment:
I think it'd be worth expanding on this slightly and explain how to work out if this bug will impact you. I hope 99% of projects don't need to worry about this at all.

Reply:
Ok, how's this?